### ISSUE TITLE
Allow to change keybindings

### ISSUE THREAD

**Issue by phaazon** [liuchengxu/vim-clap#514]
**Is your feature request related to a problem? Please describe.**

I’d like to be able to create my own keybindings. I had a look at where you define them, and tried to put them in my `ftplugin/clap.vim` file, but it doesn’t work (I have a non-us keyboard layout, so the default keybindings are pretty pointless).

**Describe the solution you'd like**
I would like to be able to remap keys via either a dictionary, or a mechanism allowing me to use `ftplugin` (like a mode-like mechanism).

**Describe alternatives you've considered**
Basically that is currently driving me off the plugin, so I’m currently using fzf.

**Comment by liuchengxu** [comment]
This should work:

```vim
autocmd FileType clap_input xxx
```

See https://github.com/liuchengxu/vim-clap/issues/66#issuecomment-640007169, also https://github.com/liuchengxu/vim-clap/issues/116#issuecomment-549816638.

**Comment by phaazon** [comment]
I’m using neovim, so I’ll try that! Thanks!

**Comment by phaazon** [comment]
Works as a charm, thanks a lot! That should be in the doc, it’s very practical! 😊

**Comment by liuchengxu** [comment]
Thanks for the remainder, always forgot to add it in the doc :).
